colpali_engine/models/qwen2/colqwen2/processing_colqwen2.py

```diff
@@ -30,14 +30,13 @@ class ColQwen2Processor(BaseVisualRetrieverProcessor, Qwen2VLProcessor):
     """
 
     visual_prompt_prefix: ClassVar[str] = (
-        "<|im_start|>user\n<|vision_start|><|image_pad|><|vision_end|>Describe the image.<|im_end|>\n"
+        "<|im_start|>user\n<|vision_start|><|image_pad|><|vision_end|>Describe the image.<|im_end|><|endoftext|>"
     )
 
     # FIXME: `query_augmentation_token` was set to hardcoded value "<pad>" in the original code used to train
     # "vidore/colqwen2-v0.1", while it should have been set to `processor.tokenizer.pad_token`.
     # TODO: Fix training script for next ColQwen2 release.
-    query_augmentation_token: ClassVar[str] = "<pad>"
```

Comment:
We have discussed about having an hardcoded `if model_name == "vidore/colqwen-v0.1"` to keep `query_augmentation_token = <pad>`. Is this still relevant?

Reply:
nah just changing to the new one cuz that's how ut was trained !